### PR TITLE
add chunker_params to borg archive info (--json)

### PR DESCRIPTION
**Pull request by leo-b**
ssia.
--leo

### CODE REVIEW

File: src/borg/archive.py [hunk 1]
```diff
@@ -399,6 +399,7 @@ def info(self):
                 'hostname': self.metadata.hostname,
                 'username': self.metadata.username,
                 'comment': self.metadata.get('comment', ''),
+                'chunker_params': self.metadata.get('chunker_params', ''),
```
Comment (ThomasWaldmann) [inline src/borg/archive.py]: looks good.
but i guess the docs about the json format also need an update about this?

Comment (leo-b) [reply src/borg/archive.py]: > but i guess the docs about the json format also need an update about this?
I hope I found the right place.. :-)

**Comment by ThomasWaldmann** [comment]
btw, the usual workflow is PR against master, review/merge, then maybe backport to 1.1 and 1.0, see the development section in our docs.

**Comment by ThomasWaldmann** [comment]
thanks, we need a forward port of this against master also, see my previous comment.

**Comment by leo-b** [comment]
> thanks, we need a forward port of this against master also, see my previous comment.
Hmm?
Did you miss anything in #3578, which you already merged on Saturday?

**Pull request merged**